<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Help_New_Project</class>
 <widget class="QMainWindow" name="Help_New_Project">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_Text_1">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>60</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>To ensure the best performance, it is highly recommended to initialize your project using an existing JSON configuration.</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Text_2">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>110</y>
      <width>451</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Loading a new ELF file directly can take up to 10 minutes. You can always update or check for changes in your ELF file once the project is open by using the &quot;Compare with ELF&quot; feature. This avoids the long initial wait time.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close_window">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>220</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Window</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::WindowClose"/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Title">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Help New Project</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>464</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
